interface/patient_file/encounter/new_form.php

```diff
@@ -258,7 +258,7 @@ function myGetRegistered($state="1", $limit="unlimited", $offset="0") {
       if($jid==0 || ($modid!=$modulerow['mod_id'])){
         if($modid!='')
         $StringEcho.= '</table></div></li>';
-      $StringEcho.= "<li><a href='JavaScript:void(0);' onClick=\"mopen('$DivId');\" >$new_category</a><div id='$DivId' ><table border='0' cellspacing='0' cellpadding='0'>";
+      $StringEcho.= "<li><a href='JavaScript:void(0);' onmouseover=\"mopen('$DivId');\" >$new_category</a><div id='$DivId' ><table border='0' cellspacing='0' cellpadding='0'>";
```

Comment:
pretty simple to debug... 👍 nice one

Reply:
added the mouseleave. onmouseleave doesn't work.